main.py

```diff
@@ -56,6 +58,7 @@ def setup(self):
         # Every create/update/delete must be synced to mongodb.
         self.circuits = {}
 
+        self.table_group = {"base": 0}
```

Comment:
Shouldn't the other table groups also be initialized here too?

Reply:
Changed, now it is initialized as `{"epl": 0, "evpl": 0}`